app/controllers/tag_controller.rb

```diff
@@ -54,6 +54,11 @@ def show
     @wikis = nodes if @node_type == 'wiki'
     @nodes = nodes if @node_type == 'maps'
     @title = params[:id]
+    
+    @all_notes = Node.where(status: 1, type: 'note')
+            .includes(:drupal_node_revision, :tag)
```

Comment:
This is good, but it may result in quite a large query, since it'll fetch the full content of each node. Could we remove the revisions include, so we just get the nodes and not all their content? Or is @all_notes going to be used elsewhere? Likewise if it's not, we could remove the order method. Thanks!